src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/OptionsTests.cs

```diff
@@ -960,6 +960,33 @@ public static void Options_RespectNullableAnnotationsDefault_FeatureSwitch(bool?
             }, arg, options).Dispose();
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public static void Options_NullabilityInfoFeatureSwitchDisabled_ReportsPropertiesAsNullable()
+        {
+            var options = new RemoteInvokeOptions()
+            {
+                RuntimeConfigurationOptions =
+                {
+                    ["System.Reflection.NullabilityInfoContext.IsSupported"] = false
+                }
+            };
+
+            RemoteExecutor.Invoke(static () =>
+            {
+                var jsonOptions = new JsonSerializerOptions { RespectNullableAnnotations = true };
+                var value = new NullableAnnotationsTests.NotNullablePropertyClass();
+                string expectedJson = """{"Property":null}""";
+
+                Assert.Null(value.Property);
+                string json = JsonSerializer.Serialize(value, jsonOptions);
+                Assert.Equal(expectedJson, json);
+                value = JsonSerializer.Deserialize<NullableAnnotationsTests.NotNullablePropertyClass>(json, jsonOptions);
+                Assert.Null(value.Property);
```

Comment:
```suggestion
                Assert.Null(value.Property);
                
                JsonTypeInfo typeInfo = jsonOptions.GetTypeInfo(typeof(NullableAnnotationsTests.NotNullablePropertyClass));
                JsonPropertyInfo propertyInfo = typeInfo.Properties.FirstOrDefault(p => p.Name == nameof(NullableAnnotationsTests.NotNullablePropertyClass.Property));
                Assert.True(propertyInfo.IsGetNullable);
                Assert.True(propertyInfo.IsSetNullable);
```

Reply:
Should we just commit this suggestion in case it case help expedite the PR. This way Erik won't have to wait for another build.